src/player/cPlayer.cpp

```diff
@@ -256,16 +256,16 @@ void cPlayer::setHouse(int iHouse)
         emblemBackgroundColor = getEmblemBackgroundColorForHouse(house);
 
         destroyAllegroBitmaps();
-        bmp_flag = createTextureFromIndexedSurfaceWithPalette(gfxdata->getSurface(BUILDING_FLAG_LARGE),232);
-        bmp_flag_small = createTextureFromIndexedSurfaceWithPalette(gfxdata->getSurface(BUILDING_FLAG_LARGE),232);
+        bmp_flag = createTextureFromIndexedSurfaceWithPalette(gfxdata->getSurface(BUILDING_FLAG_LARGE),IndTrans);
```

Comment:
Not sure what `IndTrans` means, perhaps we can rename it to something more descriptive? I didn't know there was so much transparency happening to be honest. 😅 . 

Suggestion:

`DefaultTransparency`